File: 3172-divisible-and-non-divisible-sums-difference/3172-divisible-and-non-divisible-sums-difference.cpp
class Solution {
public:
    int differenceOfSums(int n, int m) {
        vector<int>n1;
        vector<int>n2;
        for(int i=1;i<=n;i++){
            if(i%m!=0){
                n1.push_back(i);
            }
            if(i%m==0){
                n2.push_back(i);
            }
        }
        int k=accumulate(n1.begin(),n1.end(),0);
        int l=accumulate(n2.begin(),n2.end(),0);
        return k-l;
    
        
    }
};

File: 3172-divisible-and-non-divisible-sums-difference/3172-divisible-and-non-divisible-sums-difference.py
class Solution:
    def differenceOfSums(self, n: int, m: int) -> int:
        l=[]
        l2=[]
        for i in range(1,n+1):
            if(i%m!=0):
                l.append(i)
            if(i%m==0):
                l2.append(i)
        return sum(l)-sum(l2)
        
        